NeuroID/src/main/java/com/neuroid/tracker/models/NIDRemoteConfig.kt

```diff
@@ -0,0 +1,20 @@
+package com.neuroid.tracker.models
+
+import com.google.gson.annotations.SerializedName
+
+data class NIDRemoteConfig(
+    @SerializedName(value="call_in_progress")
+    val callInProgress: Boolean = true,
+    @SerializedName(value="event_queue_flush_interval")
+    val eventQueueFlushInterval: Long = 5,
+    @SerializedName(value="event_queue_flush_size")
+    val eventQueueFlushSize: Int = 1999,
+    @SerializedName(value="geo_location")
+    val geoLocation: Boolean = true,
+    @SerializedName(value="gyro_accel_cadence")
+    val gyroAccelCadence: Boolean = false,
+    @SerializedName(value="gyro_accel_cadence_time")
+    val gyroAccelCadenceTime: Long = 200L,
+    @SerializedName(value="request_timeout")
+    val requestTimeout: Long = 10) {
+}
```

Comment:
sorry for thinking about this only now but should we have all the default times either only in seconds or ms to avoid confusion?

Reply:
According to ChatGPT its okay to have these in different units so maybe not being standard is an accepted standard :)

Reply:
As long as I know which units to use and these are not changed, we should be good. Might be good to standardize. Thank for checking though. If we do change the units, please let me know. Also we should finalize any changes before we release thing thing.